CMS-Enterprise/easi-app
# [NOREF] Upgrade Okta related packages

# NOREF

## Description

- Upgrades the following packages:
  - ❗ @okta/okta-auth-js from `^6.1.0` to `^7.7.0`
  - @okta/okta-react from `^6.8.0` to `^6.9.0`
  - @okta/okta-signin-widget from `^7.18.1` to `^7.19.6`

## How to test this change

- Ensure that all tests pass in CI / locally
- Ensure that Okta login behavior functions locally as expected
  - Start the app with `scripts/dev up`
  - Ensure that you can log in with Okta (your EUA credentials) locally, and that the styling of the signin widget hasn't changed / broken.

## PR Author Checklist




## PR Reviewer Guidelines

- It's best to pull the branch locally and test it, rather than just looking at the code online!
- When approving a PR, provide a reason _why_ you're approving it
  - e.g. "Approving because I tested it locally and all functionality works as expected"
  - e.g. "Approving because the change is simple and matches the Figma design"
- Don't be afraid to leave comments or ask questions, especially if you don't understand why something was done! (This is often a great time to suggest code comments or documentation updates)
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
